docker/Dockerfile-workers

```diff
@@ -1,12 +1,28 @@
 FROM centos:8 AS builder
 
+RUN dnf -y install dnf-plugins-core \
+    && dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
```

Comment:
Since this is CentOS, you can just install EPEL with `dnf install -y epel-release`.

Reply:
done